### PR TITLE
Clean up changes left over from CF v4

### PR DESCRIPTION
**Pull request by jimmynotjim**
Short description explaining the high-level reason for the pull request

## Removals

- Removed files that are no longer used

## Changes

- Updated includes to use latest CF classes and markup examples

## Testing

1. run `gulp build`.
2. run `gulp test:acceptance`.
3. navigate throughout the site and check for errors.

## Screenshots

screen shot 2017-10-13 at 3 08 23 pm

screen shot 2017-10-13 at 3 09 49 pm

screen shot 2017-10-13 at 3 10 43 pm

## Notes

- This is only one small part of this cleanup. Breaking it up into multiple PRs to keep them from being huge.

## Checklist

* [x] PR has an informative and human-readable title
* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [x] Passes all existing automated tests
* [x] Any *change* in functionality is tested
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated
* [x] Reviewers requested with the [Reviewer tool](https://help.github.com/articles/about-pull-request-reviews/) :arrow_right:

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/related-links.html [hunk 1]
```diff
@@ -28,9 +28,14 @@ <h2 class="a-heading">
...
-    <div class="content-l content-l__main content-l__large-gutters">
+    <div class="content-l
+                content-l__main
+                content-l__large-gutters">
         <div class="content-l_col content-l_col-1-2">
-            <ul class="m-list m-list__links u-mb0">
+            <ul class="m-list
+                       m-list__unstyled
+                       m-list__links
+                       u-mb0">
```
Comment (Scotchester) [inline cfgov/jinja2/v1/_includes/related-links.html]: `u-mb0` is unnecessary with our `last-child` rule: https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/css/enhancements/typography.less#L281-L285

(Also occurs once below.)

File: cfgov/jinja2/v1/_includes/molecules/related-content.html [hunk 2]
```diff
@@ -36,7 +36,9 @@ <h2 class="a-heading">
     {{ parse_links(value.paragraph) | safe }}
 
     {% if value.links %}
-        <ul class="m-list m-list__unstyled m-list__links">
+        <ul class="m-list
+                   m-list__unstyled
+                   m-list__links">
```
Comment (Scotchester) [inline cfgov/jinja2/v1/_includes/molecules/related-content.html]: `m-list__unstyled` and `m-list__links` are redundant. See: https://github.com/cfpb/capital-framework/blob/canary/src/cf-typography/src/molecules/list.less#L6-L8

(Also occurs three times in `related-links.html`, once in `related-metadata.less`.)

**Comment by Scotchester** [comment]
Just checked this out in a browser, and sure enough, the "Clear" label is showing up right away with no text in it. It definitely seems to be something in this PR, because it works as expected (and the test passes) on master, but it's not clear to me what would be causing it.

**Comment by jimmynotjim** [comment]
Acceptance tests are passing for me. Can anyone else confirm?

**Comment by Scotchester** [comment]
OK, they're now working for me. Go forth and merge!

**Comment by jimmynotjim** [comment]
> OK, they're now working for me. Go forth and merge!

tenor

**Pull request merged**